src/components/Timer.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import '../styles/Timer.css'
+
+function Timer() {
+    const [currentValue, setCurrentValue] = useState(0);
+    const [intervalId, setIntervalId] = useState(false);
+
+    const resetTimer = () => {
+        clearInterval(intervalId);
+        setIntervalId(false);
+        setCurrentValue(0);
+    }
+
+    const startTimer = () => {
+        if (!intervalId) {
+            const startTime = Date.now();
+            const interval = setInterval(() => {
+                const value = (currentValue + Math.round((Date.now() - startTime) / 100) / 10);
```

Comment:
Consider moving any rounding to the `formatTime()` function, which already takes care of rounding for other parts of this value. That way, the logic internal to `setInterval()` can be more concise.

Reply:
This would also allow you to change the format in the future to include fractional seconds.

Reply:
Ha! I'd actually forgotten I did any rounding here. The steps were far apart. Making this change.